src/components/ComparisonTable/index.tsx

```diff
@@ -10,9 +10,11 @@ export const ComparisonTable = ({ column1, column2, children }) => {
                         <td className="text-center">
                             <strong>{column1}</strong>
                         </td>
-                        <td className="text-center">
-                            <strong>{column2}</strong>
-                        </td>
+                        {column2 && (
+                            <td className="text-center">
+                                <strong>{column2}</strong>
+                            </td>
+                        )}
```

Comment:
heads up @corywatilo @smallbrownbike that I made some tweaks to the ComparisonTable so that it can also be single column.

I prefer to use this over a standard markdown table since it looks nicer